install/deploy_management_nodes.md

```diff
@@ -503,7 +503,7 @@ be performed are in the [Deploy](#deploy) section.
         Expected output looks similar to the following:
 
         ```text
-        ncn-m001,ncn-m002,ncn-m003,ncn-s001,ncn-s002,ncn-s003,ncn-w001,ncn-w002,ncn-w003
+        ncn-m001,ncn-m002,ncn-m003,ncn-s001,ncn-s002,ncn-s003,ncn-w001,ncn-w002,ncn-w003,
```

Comment:
why is there a trailing ,??  if implying more would it be ncn-w003, ...?

Reply:
That's just how the command output looks. We could also modify the previous command to trim off the trailing comma, but `pdsh` doesn't care about the comma. I just noticed that our docs didn't show the comma in the example output, and I didn't want someone to notice that their output had the trailing comma and the example does not, and then worry that something was wrong.

I'm happy to change the command to strip the trailing comma, and then keep the example output as-is. I chose the current option only because there is leeriness to modifying commands, especially in our install documents, but not so much concern about modifying this example output.

Probably what I'll do here is change the command for the main branch PR, but for csm-1.2 I'll keep the PR as-is, since they are going to be more reluctant to make changes to commands there.,

Reply:
makes sense.  if that is the output, then I would leave it.  I was just curious why the trailing comma.